src/v2/es-utils.js

```diff
@@ -54,6 +54,37 @@ class ESClass {
       });
   }
 
+  async healthy(timeout, status) {
+    const resp = await this.client.cluster.health({
+      level: 'cluster',
+      waitForStatus: status || 'green',
+      timeout: timeout || '90s'
+    });
+    if (resp.hasOwnProperty('body') && resp.body.status !== 'red') {
+      return (resp.body);
+    }
+    const info = await this.client.info();
+    return Object.assign(resp, info);
+  }
+
+  async waitPort() {
+    const opts = {
+      host: this.env.ES_HOST,
+      port: this.env.ES_PORT,
+      output: 'silent',
+      timeout: 120000
+    };
+    await waitOn(opts)
+      .then((open) => {
+        if (!open) {
+          throw new Error(`timeout on port [${opts.port}]`);
+        }
+      })
+      .catch((err) => {
+        throw (err);
```

Comment:
`throw` is not a function call, so the `()` can be omitted

Reply:
Thx for the tip! changing it to `throw err;`

Reply:
Mixing await/async and promises makes this code more difficult to grok/parse than it should. This is roughly equivalent to but less readable than:
```
try {
if(!await waitOn(opts)) throw;
catch(err) { ... }
```

Reply:
Updated! thx!